app/views/content/fund-your-training/_why-teach.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="why-teach" aria-label="Why teach?">
+  <div class="why-teach__container">
+    <div class="why-teach__content">
+      <h2>
+        <div class="caption">Why teach?</div>
+      <h2>
+        Find out what teaching is really like
+      </h2>
+
+      <p>
+        If you’re not 100% sure if teaching is the right career for you,
+        you can find out more about what being a teacher is all about.
+      </p>
+
+      <ul>
+        <li><%= link_to("Experience life in the classroom", "/get-school-experience") %></li>
+        <li><%= link_to("Meet teachers and get expert advice", "/events") %></li>
```

Comment:
Do we want to link to events in both the questions section and the why teach section?

Reply:
I'd suggest linking to the blog here instead - 'Read teachers' stories' or similar